sinks_test.go

```diff
@@ -166,6 +172,60 @@ func TestSHA256Sum(t *testing.T) {
 	}
 }
 
+func TestSliceSink(t *testing.T) {
+	t.Parallel()
+	input := Echo("testdata/multiple_files/1.txt\ntestdata/multiple_files/2.txt\ntestdata/multiple_files/3.tar.zip\n")
```

Comment:
Can you turn this into a table-driven test, with named test cases, using `t.Run()`?